examples/va/hci/service-values.yaml

```diff
@@ -51,7 +51,7 @@ data:
       replicas: 3
       customServiceConfig: |
         [DEFAULT]
-        enabled_share_protocols=cephfs
+        enabled_share_protocols=nfs
```

Comment:
-1: I understand this is about "updates", but this would result in breaking VA1 (unless there's something I missed in the entire updates context).
@lkuchlan not sure you can give me more context, but I think this won't work.
In addition, VA/HCI doesn't deploy ganesha (it's in the unidelta DT), so I'm not in favor of merging this change

Reply:
Thanks, I thought it was an update job because of the ci-framework-baremetal-rhel9-va-hci-update-trunk-patches-base job. 
Where should the change be applied?

Reply:
Maybe there a no jobs in ci-framework which deploys manila with ganesha, as there are no public jobs which use uni-delta.

Reply:
@tosky agreed

@lkuchlan think we should clone the base job into a downstream/component level update job and enable cephfs-nfs there instead, like we do for uni-delta.. i'd suggest doing this for completeness.

However, I think the base update job testing manila with Native CephFS is totally fine.. the point of updates testing would be to see the consequences of updating the OpenStack operators, and swapping out service container images... There shouldn't be a huge difference testing with CephFS-NFS since the Ceph cluster in the deployment isn't touched during the update process.

Reply:
So let's leave it as it is with Native CephFS. I'll just adjust the job to run the Manila Tempest tests.